pabi_procurement/__openerp__.py

```diff
@@ -30,10 +30,13 @@
         "stock_operating_unit"
```

Comment:
Please add "purchase_double_validation" here, then you don't need to manual configure it.